qa/os/src/test/java/org/elasticsearch/packaging/util/DockerRun.java

```diff
@@ -75,6 +76,13 @@ public DockerRun uid(Integer uid, Integer gid) {
         return this;
     }
 
+    public DockerRun memory(String memoryLimit) {
```

Comment:
The intention here was to then specifically add a docker packaging test where we set the container memory limit and test that the auto configured heap is as we'd expect.

Reply:
Are we going to add such a test?

Reply:
To close the loop here - @rjernst wrote a basic, and I've fleshed it out and debugged it locally. All good now, I think.